components/about/Team.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="about-team">
+    <AboutSvgWavyBump top />
+    <div class="about-team-content">
+      <v-container>
+        <v-row>
+          <h2>Meet the Team</h2>
+        </v-row>
+        <v-row>
+          <v-col cols="12" sm="12" md="6" lg="6" v-for="lead in leads" :key="lead.name">
+            <AboutTeamMember
+              :github-handle="lead.githubHandle"
+              :twitter-handle="lead.twitterHandle"
+              :name="lead.name"
+              :role="lead.role"
+              :custom-pic="lead.customPic"
+            />
+          </v-col>
+          <v-col sm="12" md="6" lg="6">
+            <div class="about-team-more-contributors">
+              <div>
+                <h3>Plus many more members who have lent us a helping hand!</h3>
+                <br />
+                <h3>Find them on our <a href="https://github.com/ourjapanlife/findadoc-frontend">front end</a> and <a href="https://github.com/ourjapanlife/findadoc-localization">localization</a> Github contributors pages</h3>
+              </div>
+            </div>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col sm="12">
+            <div class="about-team-join">
+              <h3>Does Find a Doc’s mission spark your interests?</h3>
+              <h3>Join us over on GitHub to see how you can contribute!</h3>
+              <h3><a href="https://github.com/ourjapanlife">✋ I want to join!</a></h3>
+            </div>
+          </v-col>
+        </v-row>
+      </v-container>
+    </div>
+    <AboutSvgWavyBump />
+  </div>
+</template>
+<script>
+import { leads } from "../../data/contributors.json";
+
+export default {
+  data() {
+    return {
+      leads: leads
+    };
+  }
+};
+</script>
+<style>
+  .about-team {
+    color:  white;
+  }
+
+  .about-team a {
+    color:  white;
+  }
+
+  .about-team h2 {
+    font-family: Montserrat;
```

Comment:
We should follow consistent font types throughout the app. Is this meant to be our secondary font type or a one off?

Reply:
This is meant to match Teresa's design, in which the large headers are often, but not always, Monserrat. I agree that it'd be great to migrate all these up to a global when we get SCSS in.

Reply:
I set Monserrat as the default font for headings across the site in `assets/variable.scss` in a [previous PR](https://github.com/ourjapanlife/findadoc-frontend/pull/248). It should work for you if you pulled the latest changes from Main.

Reply:
@theyokohamalife Ah yeah, I see them. These are set only to the Vue header classes (ie `.text-h3`) and not the the actual elements. I think we should follow-on to that and set the element defaults as well, but also that should be done separately and examined globally.

I think we should leave this for now, until we do the above - I'd rather not just replace one extra line we'll need to delete later with another, as `<h3 class="text-h3"` is redundant.